tests/positive/Format.juvix

```diff
@@ -65,19 +61,7 @@ t5 : String := "a" M., "b" M., "c", "d";
 t2 : String := "a", "b", "c", "d";
 
 -- comma chain does not fit in a line
```

Comment:
The list needs to be made longer so that we force linebreaks

Reply:
done